ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/ConnectorBindingsRhino.Receive.cs

```diff
@@ -650,6 +640,42 @@ void FlattenConvertedObject(object item)
           bakedCount++;
           break;
 
+        case Group o: // this is a GIS object
```

Comment:
Are we sure this is the only case this would happen?

Would this explode if the converter returns a group that has no attributes for example?